dtls/server-dtls-callback.c

```diff
@@ -81,11 +80,33 @@ int my_IORecv(WOLFSSL* ssl, char* buff, int sz, void* ctx)
         shared->rxSz -= recvd;
         memcpy(shared->rxBuf, &shared->rxBuf[recvd], shared->rxSz);
     }
```

Comment:
I just noticed that there is this `rxSz` in the `shared` struct that is not set anywhere. Can you remove it to simplify the example?